docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -240,6 +240,17 @@ With a AWS IAM key pair
       "password": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
     }'
 
+Reference as Jinja templated fields
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To render the AWS credentials using Jinja template reference ``AWS_ACCESS_KEY_ID`` as ``login`` and ``AWS_SECRET_ACCESS_KEY`` as ``password``
+
+  .. code-block:: bash
+
+    echo "AWS_ACCESS_KEY_ID={{ conn.connection_id.login }}"
+    echo "AWS_SECRET_ACCESS_KEY={{ conn.connection_id.password }}"
```

Comment:
Just for a reference it would only works in case if you use [long-term credentials](https://docs.aws.amazon.com/sdkref/latest/guide/access-iam-users.html) and it stored into the login / password fields

Reply:
Actually I stored the values directly into `AWS Access Key ID` and `AWS Secret Access Key` and it worked with `login` and `password` template fields. 
![image](https://github.com/apache/airflow/assets/31843161/a9f225ff-8ffa-41b9-8190-2f2a50118a97)

Reply:
Yeah, in the UI it have this name because it relabelling in the Hook

https://github.com/apache/airflow/blob/5f4d2b5dee2e98a05124865c6fa855bc26d0af26/airflow/providers/amazon/aws/hooks/base_aws.py#L820-L823

All connections regardless of the conn_type have the same structure because you always retrieve `airflow.models.connection.Connection` object

https://github.com/apache/airflow/blob/f24e5199c793c48a2f8cf5fe6bb2699cd8283196/airflow/models/connection.py#L63-L84

Reply:
Anyway it might be a good point to mentioned about relabelling and this affect not only an AWS Connection

Reply:
Info added